src/tools/wasm-interp.cc

```diff
@@ -39,6 +40,9 @@
 using namespace wabt;
 using namespace wabt::interp;
 
+using ExportMap = std::map<std::string, Extern::Ptr>;
+using Registry = std::map<std::string, ExportMap>;
```

Comment:
Can/should we use unordered_map here?

Reply:
I am new to the game (last time I programmed C++ was 5-8 years ago), but I guess yes, does not make any difference.
I am going to update this.